src/ucp/proto/proto_multi.c

```diff
@@ -200,6 +203,9 @@ ucs_status_t ucp_proto_multi_init(const ucp_proto_multi_init_params_t *params,
         mpriv->max_frag_sum += lpriv->max_frag;
         lpriv->weight_sum    = weight_sum;
         lpriv->max_frag_sum  = mpriv->max_frag_sum;
+        lpriv->opt_align     = lane_perf->opt_align;
```

Comment:
let's remove lane_perf->opt_align (remove field opt_align from tl_perf struct) and keep only lpriv->opt_align; initialize lpriv->opt_align directly from ucp_proto_multi_get_lane_opt_align() result